distribution/docs/src/main/resources/_contents/_catalog-contents/extending-catalog-contents.adoc

```diff
@@ -3133,6 +3133,259 @@ Supports "asc" and "desc" using colon as delimiter.
 
 The OpenSearch source does not provide a Connected Source interface.
 
+==== Confluence Source
+
+The Confluence source provides a Federated Source to retrieve pages/comments/attachments from a Confluence REST API  and turns the results into Metacards the system can use.
+The Confluence source does provide a Connected Source interface but its functionality has not been verified.
+
+Confluence Source has been tested against the following versions of Confluence with REST API v2
+
+* Confluence 1000.444.5 (Cloud)
+* Confluence 5.10.6 (Server)
+* Confluence 5.10.7 (Server)
+
+===== Installing and Uninstalling
+
+The Confluence source can be installed and uninstalled using the normal processes described in the Configuring ${branding} section.
```

Comment:
We should say if it is installed by default and then guide readers through the process.